test/Microsoft.AspNetCore.SignalR.Tests/EndToEndTests.cs

```diff
@@ -260,7 +260,7 @@ public static IEnumerable<object[]> MessageSizesData
         [MemberData(nameof(MessageSizesData))]
         public async Task ConnectionCanSendAndReceiveDifferentMessageSizesWebSocketsTransport(string message)
         {
-            using (StartLog(out var loggerFactory, LogLevel.Trace, testName: $"ConnectionCanSendAndReceiveDifferentMessageSizesWebSocketsTransport_{message.Length}"))
+            using (StartLog(out var loggerFactory, LogLevel.Information, testName: $"ConnectionCanSendAndReceiveDifferentMessageSizesWebSocketsTransport_{message.Length}"))
```

Comment:
Leave it as Trace

Reply:
It floods the logs with the large message